x-pack/plugins/monitoring/server/lib/elasticsearch/nodes/get_nodes/get_nodes.js

```diff
@@ -55,6 +55,7 @@ export async function getNodes(req, esIndexPattern, clusterStats, shardStats) {
     index: esIndexPattern,
     size: config.get('xpack.monitoring.max_bucket_size'),
     ignoreUnavailable: true,
+    rest_total_hits_as_int: true,
```

Comment:
We're using `hits.total` here, but we could also change to checking `hits.hits.length > 0` again. In this particular case, we are using field collapsing, so `hits.total` is inherently incorrect anyway (an overestimate), so it was really just a matter of simplicity to check `hits.total` for not being `0`.

Reply:
Sounds good, will adjust